pymoto/modules/assembly.py

```diff
@@ -223,6 +223,42 @@ def _prepare(self, domain: DomainDefinition, *args,
         super()._prepare(domain, self.KE, *args, **kwargs)
 
 
+def ConsistentMassEq(domain: DomainDefinition, ndof: int, MP: float = 1.0):
+    """
+    Calculates Consistent mass element matrix, or its equivalents (e.g. Damping, Capacitance)
+
+    Args:
+        domain: The domain the element matrix has to be determined for
+        ndof: Amount of dofs per node
+            Mass, Damping: ndof = domain.dim
+            Else: ndof = 1
+        MP: Material property to use in the element matrix
+            Mass: Density (rho)
+            Else: Thermal capacity, Damping parameter
+
+    Returns:
+        Consistent mass equivalent element matrix
+    """
+
+    consEl = np.zeros((domain.elemnodes*ndof, domain.elemnodes*ndof))
+
+    # Numerical integration
+    siz = domain.element_size
+    w = np.prod(siz[:domain.dim]/2)
```

Comment:
These weights originate from the choice of integration scheme. It might be more insightful to program some integration scheme classes, such that it is clear where this is coming from. E.g. IntegrationScheme and then GaussLegendreQuadrature(IntegrationScheme) which can be called with parameter number of points.

Reply:
I think that such an object would be overkill for the current application. For now we could do with just a comment